man/bootctl.xml

```diff
@@ -111,6 +111,17 @@
         for the meaning of that variable.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>reboot-to-firmware</option> <optional><replaceable>BOOL</replaceable></optional></term>
+
+        <listitem><para>Query or set the "Reboot-Into-Firmware-Setup" flag of the EFI firmware. Takes a
+        boolean argument which controls whether to show the firmware setup on next system reboot. If the
+        argument is omitted shows the current status of the flag, or whether the flag is supported. This
+        controls the same flag as <command>systemctl reboot --firmware-setup</command>, but is more
+        low-level and allows setting the flag independently from actually requesting a
+        reboot.</para></listitem>
+      </varlistentry>
```

Comment:
Please mention the return value here and maybe alos in "Return value" section below too. Returning 1 or 2 on something-that-is-a-kind-of-sucess it a bit unexpected (though very reasonable).